### PR TITLE
Sweeper changes

### PR DESCRIPTION
**Pull request by cakeben**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: acr-build.yml [hunk 1]
```diff
@@ -14,13 +14,23 @@ steps:
...
+
   - id: push-images
     push:
       - "{{.Run.Registry}}/ccd/logstash:latest"
       - "{{.Run.Registry}}/ccd/logstash:ccd-latest"
       - "{{.Run.Registry}}/ccd/logstash:ccd-{{.Run.ID}}"
       - "{{.Run.Registry}}/ccd/logstash:probate-latest"
       - "{{.Run.Registry}}/ccd/logstash:probate-{{.Run.ID}}"
+      - "{{.Run.Registry}}/ccd/logstash:divorce-latest"
+      - "{{.Run.Registry}}/ccd/logstash:divorce-{{.Run.ID}}"
```
Comment (mario-paniccia) [inline acr-build.yml:32]: why you need this?
The ccd divorce LS is already published under `ccdprobate-logstash`

Comment (cakeben) [reply acr-build.yml:32]: I'm not sure how @linusnorton's commit got in my PR :D

Comment (mario-paniccia) [reply acr-build.yml:32]: that should have been done as part of https://tools.hmcts.net/jira/browse/RDM-10064
let me check hold on

Comment (mario-paniccia) [reply acr-build.yml:32]: ignore this comment, I thought we already had modified this file to publish the various ccd LS instances in ACR as part of https://tools.hmcts.net/jira/browse/RDM-10064 as was requested in that ticket, but I found out that wasn't done instead

Comment (mario-paniccia) [reply acr-build.yml:32]: but yes Linus apparently has already done the change for Divorce, but we should call that `ccddivorce-logstash`

File: packer_images/sweeper_logstash.conf.in [hunk 2]
```diff
@@ -7,54 +7,165 @@ input {
         jdbc_validate_connection => true
         jdbc_driver_library => "/usr/share/logstash/postgresql-42.2.2.jar"
         jdbc_driver_class => "org.postgresql.Driver"
+        jdbc_default_timezone => "UTC"
```
Comment (mario-paniccia) [inline packer_images/sweeper_logstash.conf.in:10]: beware I think `created_date` and `last_modified` in CCD are not UTC. Those are local datetimes
I wouldn't change this now but tackle this as a separate ticket to limit risks

**Pull request closed without merging**